### PR TITLE
native: cleanup comments in Dgeqrf and Ilaenv

### PR DESCRIPTION
**Pull request by vladimir-ch**

### CODE REVIEW

File: native/dgeqrf.go [hunk 1]
```diff
@@ -43,20 +43,21 @@ func (impl Implementation) Dgeqrf(m, n int, a []float64, lda int, tau, work []fl
...
 	var nx int // Use unblocked (unless changed in the next for loop)
 	iws := n
 	ldwork := nb
-	// Only consider blocked if the suggested number of blocks is > 1 and the
-	// number of columns is sufficiently large.
-	if nb > 1 && k > nb {
-		// nx is the crossover point. Above this value the blocked routine should be used.
+	// Only consider blocked if the suggested block size is > 1 and the
+	// number of rows or columns is sufficiently large.
+	if 1 < nb && nb < k {
```
Comment (vladimir-ch) [inline native/dgeqrf.go]: I think this order is clearer but I can drop it if considered unnecessary. My main goal was to make the comments accurate.

File: native/dgeqrf.go [hunk 2]
```diff
@@ -67,7 +68,7 @@ func (impl Implementation) Dgeqrf(m, n int, a []float64, lda int, tau, work []fl
 	}
 	// Compute QR using a blocked algorithm.
 	var i int
-	if nb >= nbmin && nb < k && nx < k {
+	if nbmin <= nb && nb < k && nx < k {
```
Comment (vladimir-ch) [inline native/dgeqrf.go]: Also here.

File: native/ilaenv.go [hunk 3]
```diff
@@ -5,10 +5,11 @@
...
 
 // Ilaenv returns algorithm tuning parameters for the algorithm given by the
-// input string. ispec specifies the parameter to return.
-//  1: The optimal block size
-//  2: The minimum block size for which the algorithm should be used.
-//  3: The crossover point below which an unblocked routine should be used.
+// input string. ispec specifies the parameter to return:
+//  1: The optimal block size for a blocked algorithm.
+//  2: The minimum block size for a blocked algorithm.
+//  3: The block size of unprocessed data at which a blocked algorithm should
```
Comment (vladimir-ch) [inline native/ilaenv.go]: Suggestions for better wording are welcome.

File: native/dgeqrf.go (hunk 1 above)
Comment (kortschak) [inline native/dgeqrf.go]: I like this.

File: native/ilaenv.go (hunk 3 above)
Comment (kortschak) [inline native/ilaenv.go]: This seems fine. Maybe change "switch" to "crossover" since that have a specific use in this context.

**Comment by kortschak** [comment]
LGTM

File: native/ilaenv.go (hunk 3 above)
Comment (vladimir-ch) [inline native/ilaenv.go]: Done.

**Pull request merged**